### PR TITLE
new UI side telemetry events for variable view

### PR DESCRIPTION
**Pull request by IanMatthewHuff**
For #4649 

-   [x] Pull request represents a single change (i.e. not fixing disparate/unrelated things in a single PR).
-   [x] Title summarizes what is changing.
-   [x] Has a [news entry](https://github.com/Microsoft/vscode-jupyter/tree/main/news) file (remember to thank yourself!).
-   [ ] Appropriate comments and documentation strings in the code.
-   [ ] Has sufficient logging.
-   [x] Has telemetry for enhancements.
-   [ ] Unit tests & system/integration tests are added/updated.
-   [ ] [Test plan](https://github.com/Microsoft/vscode-jupyter/blob/main/.github/test_plan.md) is updated as appropriate.
-   [ ] [`package-lock.json`](https://github.com/Microsoft/vscode-jupyter/blob/main/package-lock.json) has been regenerated by running `npm install` (if dependencies have changed).

### CODE REVIEW

**Comment by IanMatthewHuff** [comment]
The JupyterVariables interface has some telemetry. But on checking added some UI side telemetry for the new variable view.

File: src/client/datascience/variablesView/variableView.ts [hunk 1]
```diff
@@ -82,6 +82,7 @@ export class VariableView extends WebviewViewHost<IVariableViewPanelMapping> imp
         this.dataViewerChecker = new DataViewerChecker(configuration, appShell);
     }
 
+    @captureTelemetry(Telemetry.NativeVariableViewLoaded)
```
Comment (IanMatthewHuff) [inline src/client/datascience/variablesView/variableView.ts:85]: This seemed extra useful as we are not using retain context on the webview. Means that it gets loaded fairly often. Here to make sure that this is not taking too long.

File: src/client/datascience/variablesView/variableView.ts [hunk 2]
```diff
@@ -182,6 +184,9 @@ export class VariableView extends WebviewViewHost<IVariableViewPanelMapping> imp
             const response = await this.variables.getVariables(args, this.notebookWatcher.activeNotebook);
 
             this.postMessage(InteractiveWindowMessages.GetVariablesResponse, response).ignoreErrors();
+            sendTelemetryEvent(Telemetry.VariableExplorerVariableCount, undefined, {
```
Comment (IanMatthewHuff) [inline src/client/datascience/variablesView/variableView.ts:187]: This is same event in the same spot as with the old webviews, lets us see how many variables we are fetching.

**Review by rchiodo — APPROVED** [review]

**Review by DavidKutu — APPROVED** [review]

**Review by joyceerhl — APPROVED** [review]

**Comment by IanMatthewHuff** [comment]
Failure looks like the kernel switch flake issue.

**Pull request merged**